PowerEditor/visual.net/notepadPlus.sln

```diff
@@ -0,0 +1,54 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.852
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Notepad++", "notepadPlus.vcxproj", "{FCF60E65-1B78-4D1D-AB59-4FC00AC8C248}"
+	ProjectSection(ProjectDependencies) = postProject
+		{FBE04237-9C7B-4973-9C60-505975998B39} = {FBE04237-9C7B-4973-9C60-505975998B39}
+	EndProjectSection
+EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "libSciLexer", "..\..\scintilla\win32\libSciLexer.vcxproj", "{FBE04237-9C7B-4973-9C60-505975998B39}"
```

Comment:
Rename libScilexer to Scintilla.
The reason is Scintilla is the official name of the project, so it's normal to use its original name as the project name.

Reply:
@donho 
This one is derived from https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/scintilla/win32/SciLexer.vcxproj
and see also https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/scintilla/win32/scintilla.mak#L16-L18

we have now 4 build components there:

> COMPONENT=$(DIR_BIN)\Scintilla.dll
> LEXCOMPONENT=$(DIR_BIN)\SciLexer.dll
> LIBSCI=$(DIR_BIN)\libscintilla.lib
> LIBLEX=$(DIR_BIN)\libscilexer.lib

Scintilla is the build without the lexer components and SciLexer the one with. Therefore also the corresponding naming of the static libs. We used the SciLexer.dll before.
Should I still rename it?

Reply:
@chcg 
So every update, we have to transform from `SciLexer.vcxproj` to `libSciLexer.vcxproj`, right?
Why don't we just keep `SciLexer.vcxproj` name with libSciLexer content then?

Reply:
@donho Yes, that is necessary, if changes are done to SciLexer.vcxproj by scintilla upstream versions.
I could also transform SciLexer.vcxproj directly. Depends if you want to be able to still build the scilexer.dll via Visual Studio and what is easier for you to handle.

Reply:
@chcg Hmm... Since Scintilla is static lib and linked in Notepad++ directly, I see no reason to build it as dll now. Let's use  SciLexer.vcxproj to build static lib directly.

BTW, have we any (proximal) date for the official release of Boost 1.76?

Reply:
See https://www.boost.org/development/index.html 14.Apr

Reply:
Thank you!

Reply:
@chcg 
I don't think there'll be some change in the regexpr's headers from boost 1.76 beta to 1.76 release.
I would say let's merge it into master now , so the this new structure can be checked from now on - we can always check/integrate regexpr headers after boost 1.76 official release. What do you think?